scripts/make_msd_index.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eux
+
+MSD_FOLDER=/msd
+
+find $MSD_FOLDER -name *.h5 > msd-index.txt 2>/dev/null
+
+shuf msd-index.txt | head -n10000 > msd-sample.txt
```

Comment:
Can you set a seed/make this deterministic?

Reply:
Yep!